tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist = py36, py37, py37-win
+# https://tox.readthedocs.io/en/latest/config.html#conf-requires
+# Ensure pip is new so we can install manylinux wheel
+requires = pip >= 19.0.0
+           virtualenv >= 16.0.0
+
+[testenv]
+deps =
```

Comment:
The old travis workflow used https://github.com/IDR/idr-py/blob/v0.4.0/test-requirements.txt

Is it possible to avoid duplicating the test/dev dependencies in two places? Also this list of dependencies is different from https://github.com/IDR/idr-py/blob/v0.4.0/requirements.txt and https://github.com/IDR/idr-py/blob/v0.4.0/test-requirements.txt

Reply:
flake8 has been replaced by pre-commit
For the usage of requirements file vs dependencies in tox.ini, we do not use the requirements elsewhere this is no different.
test-requirements should probably be removed now since it was needed for travis